.github/workflows/macos.yml

```diff
@@ -29,18 +27,14 @@ jobs:
       - name: Build kDrive desktop
         run : ./infomaniak-build-tools/macos/build-ci.sh
 
-      - name: Execute common tests
-        run : ./kDrive_test_common
-        working-directory: ${{env.bin-directory}}
-        continue-on-error: true
-
-      - name: Execute parms tests
-        run : ./kDrive_test_parms
-        working-directory: ${{env.bin-directory}}
-        continue-on-error: true
+      - name: Grant tests script execute permission
+        run : chmod +x ./infomaniak-build-tools/run-tests.sh
+      - name: Execute tests
+        run : ./infomaniak-build-tools/run-tests.sh
 
       - name: Get the path to kDrive-logdir
         run : echo "logdir_path="$(find /private/var/folders/ -name kDrive-logdir 2>/dev/null)"" >> $GITHUB_ENV
+        if  : success() || failure()
```

Comment:
So, it runs in any case? Does it?

Reply:
It would run whether the build and tests succeeded or failed, but not if the workflow was cancelled